portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -62,7 +62,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     for (Entity entity : results.asIterable()) {
       String id = KeyFactory.keyToString(entity.getKey());
       Map<String, String> commentAttributes = new HashMap<>();
-      commentAttributes.put("comment", (String) entity.getProperty("comment"));
+      commentAttributes.put("commentText", (String) entity.getProperty("commentText"));
```

Comment:
Wasn't this part of a previous PR?

Reply:
Yes, sorry I'm still trying to figure out exactly what to do with these branches and that's why I hadn't sent these links to the team chat yet. I think something went wrong when creating the branches.

Reply:
I think I will close these pull requests and try the branching again.